extension/chrome/elements/compose-modules/compose-storage-module.ts

```diff
@@ -55,14 +55,16 @@ export class ComposeStorageModule extends ViewModule<ComposeView> {
   }
 
   public collectAllAvailablePublicKeys = async (senderEmail: string, senderKi: KeyInfo, recipients: string[]): Promise<CollectPubkeysResult> => {
-    const contacts = await ContactStore.get(undefined, recipients);
+    const contacts = await ContactStore.getEncryptionKeys(undefined, recipients);
```

Comment:
I made sure `getEncryptionKeys()` returns an entry per each input recipient so that missing emails are guaranteed to make their way to `emailsWithoutPubkeys`. I suspect they may get lost with the previous version of the code if the contact store didn't have an entry for them

Reply:
Got it